### PR TITLE
Tutorial page

### PR DESCRIPTION
**Pull request by sevresbabylone**
image
image

- Changes the structure of tutorial config to be divided into 4 parts

1. Introduction section with an array of slides which displays a title, description and 1 image (if necessary)
2. A section to show what Swipe right represents
3. A section to show what Swipe left represents
4. A trial section that needs to be passed

- taken_tutorial status of User is set to true if the user clears the trial
- Page dots to jump through tutorial sections, does not allow user to skip sections
image
- adds keydown events for left and right swipe
- disallow viewing tutorial if user is not logged in (to keep things simple)

Note that this tutorial section is currently incomplete and needs to account for ImageSwipe, other potential widgets.

https://swipes-for-science-demo.netlify.app/#/tutorial

### CODE REVIEW

**Comment by sevresbabylone** [comment]
@kylesome would be great if you could help me with this! no rush though

File: src/components/Tutorial/IntroductionSlides.vue [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.tutorial__steps {
+  position: relative;
+  overflow-y: scroll;
+  scroll-snap-type: y mandatory;
+  -ms-overflow-style: none; /* Internet Explorer 10+ */
```
Comment (kylesome) [inline src/components/Tutorial/IntroductionSlides.vue:74]: Wondering if something like PostCSS would handle these ones?

File: src/components/Widgets/WordSwipe/WordSwipe.vue [hunk 2]
```diff
@@ -72,6 +85,7 @@ export default {
     const evaluateVote = async response => {
       if (props.allSamples[currentSampleId.value].actualValue === response)
         return 1;
+      return 0;
```
Comment (kylesome) [inline src/components/Widgets/WordSwipe/WordSwipe.vue:88]: looks to be declared as an async function but can't see any async calls inside `evaluateVote()`?

**Review by kylesome — COMMENTED** [review]
nice job! 👍

File: src/components/Tutorial/IntroductionSlides.vue (hunk 1 above)
Comment (sevresbabylone) [reply src/components/Tutorial/IntroductionSlides.vue]: i decided to remove it since we weren't supporting ie10 in any case.

**Pull request merged**